### PR TITLE
merge master and resolve conflicts

### PR DESCRIPTION
**Pull request by UnorderedSigh**
Merges master to experimental again. This is a draft because I haven't tested it yet. Actually, it's mostly to compare against #30 and see what went wrong.

### CODE REVIEW

File: source/PreferencesPanel.cpp [hunk 1]
```diff
@@ -548,7 +548,6 @@ void PreferencesPanel::DrawSettings()
 		STATUS_OVERLAYS_ENEMY,
 		STATUS_OVERLAYS_NEUTRAL,
 		"Show missile overlays",
-		"Show asteroid scanner overlay",
```
Comment (Zitchas) [inline source/PreferencesPanel.cpp]: Did this setting get removed?

**Comment by Zitchas** [comment]
This has no conflicts, and aside from that one preference panel line about asteroid scanner overlay, I think it looks good. I think this could be merge instead of #30 . If you agree just switch this to ready.

**Comment by UnorderedSigh** [comment]
I merged master again, and it seems to have added a ton of commits, without changing the branch at all.

Also, I added the missing line, and reverted one accidental change (space removed from a comment).

**Comment by UnorderedSigh** [comment]
I suggest you merge *without* squashing this time.

**Comment by Zitchas** [comment]
So, the "Create a merge commit" option, right?

**Comment by UnorderedSigh** [comment]
> So, the "Create a merge commit" option, right?

Yes.

**Pull request merged**